src/main/java/uk/gov/hmcts/reform/ccd/document/am/controller/endpoints/CaseDocumentAmController.java

```diff
@@ -70,13 +67,7 @@ public CaseDocumentAmController(DocumentManagementService documentManagementServ
     public ResponseEntity<Object> deleteDocumentbyDocumentId(
         @PathVariable("documentId") UUID documentId,
         @Valid @RequestParam(value = "permanent", required = false, defaultValue = "false") Boolean permanent) {
-        ResponseEntity responseEntity = documentManagementService.getDocumentMetadata(documentId);
-        if (documentManagementService.checkUserPermission(responseEntity, documentId, Permission.UPDATE)) {
-            return  documentManagementService.deleteDocument(documentId, permanent);
-
```

Comment:
why it removed, we are not going to check permission before calling delete API.There are some new requirement?

Reply:
Yes, as per the agreed approach discussed with Jon, the Delete and Patch operation will be authorised at service level and new authorization table will be implemented in next sprint.